fastlane/Fastfile

```diff
@@ -95,6 +95,7 @@ end
 
 desc 'Builds the app using the App Store target but without code signing, just to check the target builds'
 lane :test_app_store_build do | options |
+  configure_apply
```

Comment:
Above I wrote that coping the demo secrets in CI:

> is unnecessary now, because the build phase script now takes care of falling back to the demo credentials.

On the other hand, because this step builds the App Store target in Release mode and the script  fails if the secrets can't be found on a Release build.